src/Query/TaskGroupingTree.ts

```diff
@@ -73,13 +73,13 @@ export class IntermediateTaskGroups {
         const root = new TaskGroupingTreeNode(tasks);
 
         let currentTreeLevel = [root];
-        for (const grouping of groupings) {
+        for (const grouper of groupers) {
             const nextTreeLevel = [];
             for (const currentTreeNode of currentTreeLevel) {
                 for (const task of currentTreeNode.values) {
-                    // Get properties of a task for the grouping
+                    // Get properties of a task for the grouper
```

Comment:
`properties` again here. Probably if the next comment is taken into account the code will be self documented and these 2 comment lines will be unnecessary.